BLSQ/iaso
# IA-2711: Change Requests - Highlight groups and types changes

Explain what problem this PR is resolving

Related JIRA tickets : [IA-2711](https://bluesquare.atlassian.net/browse/IA-2711)

## Self proofreading checklist



## Changes

Added a new component to separate existing group and highlight new added groups to review!


## How to test

From org units **Review change proposals** page, on the action column, open an org unit to review changes and on the Groups line, when there is any new added group, it will appear with **red** color in the column **Modified version** .

e.g: 
![Highlight-added-group-Review-change-proposals](https://github.com/BLSQ/iaso/assets/6383219/43c21b44-553e-4e40-81c6-90cf5b93591d)


## Print screen / video

https://github.com/BLSQ/iaso/assets/6383219/8c5a8334-e980-4c25-8431-5d8f9d933bb9




[IA-2711]: https://bluesquare.atlassian.net/browse/IA-2711?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
